lua/nvim-lsp-installer/servers/foam_ls/init.lua

```diff
@@ -0,0 +1,16 @@
+local server = require "nvim-lsp-installer.server"
+local path = require "nvim-lsp-installer.path"
+local npm = require "nvim-lsp-installer.installers.npm"
+
+return function(name, root_dir)
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        homepage = "https://github.com/FoamScience/foam-language-server",
+        languages = { "foam", "OpenFOAM" },
+        installer = npm.packages { "foam-language-server" },
+        default_options = {
+            cmd = { path.concat { root_dir, "node_modules", "foam-language-server", "bin", "foam-ls" }, "--stdio" },
```

Comment:
By providing the PATH to the executable, we can rely on the default cmd in `lspconfig` instead. With npm installations, there's a shorthand like this:

```suggestion
            cmd_env = npm.env(root_dir),
```

Reply:
I swear I did this yesterday lol but oh well!

Reply:
git never lies 😛 !